app/manage.py

```diff
@@ -7,4 +7,20 @@
 
     from django.core.management import execute_from_command_line
 
-    execute_from_command_line(sys.argv)
+    args = sys.argv
+
+    execute_from_command_line(args)
+
+    if args[1] == "makemigrations":
```

Comment:
Actually it's executing the command first (line 12) and then checking what command was executed and then executing another command after that